src/next/azext_next/__init__.py

```diff
@@ -25,8 +26,7 @@ def __init__(self, cli_ctx=None):
         from azure.cli.core.commands import CliCommandType
         next_custom = CliCommandType(
             operations_tmpl='azext_next.custom#{}')
-        super(NextCommandsLoader, self).__init__(cli_ctx=cli_ctx,
-                                                 custom_command_type=next_custom)
+        super().__init__(cli_ctx=cli_ctx, custom_command_type=next_custom)
```

Comment:
Why do we need to modify here?

Reply:
> Why do we need to modify here?

To fix a suggestion reported by `azdev style next`.